lib/View/BatchDeleteView.js

```diff
@@ -5,6 +5,7 @@ class BatchDeleteView extends View {
         super(name);
 
         this._type = 'BatchDeleteView';
+        this._enabled = true;
```

Comment:
Also the batch-delete actions is disablable by using the batchActions method on the list view.